ui/app/src/views/ViewDashboardList.tsx

```diff
@@ -16,19 +16,41 @@ import {
   AccordionDetails,
   AccordionSummary,
   Box,
+  Button,
   CircularProgress,
   Container,
+  IconButton,
   Stack,
   Typography,
 } from '@mui/material';
+
+import { MouseEvent, useCallback, useState } from 'react';
 import { DashboardResource } from '@perses-dev/core';
 import { ErrorAlert, ErrorBoundary } from '@perses-dev/components';
 import ChevronDown from 'mdi-material-ui/ChevronDown';
 import FolderPound from 'mdi-material-ui/FolderPound';
+import DeleteOutline from 'mdi-material-ui/DeleteOutline';
+import { useNavigate } from 'react-router-dom';
 import { useDashboardList } from '../model/dashboard-client';
+import { ProjectModel } from '../model/project-client';
 import DashboardList from '../components/DashboardList';
+import DeleteProjectDialog from '../components/DeleteProjectDialog/DeleteProjectDialog';
+import AddProjectDialog from '../components/AddProjectDialog/AddProjectDialog';
 
 function RenderDashboardList() {
+  const [openDeleteProjectDialog, setOpenDeleteProjectDialog] = useState<boolean>(false);
```

Comment:
What do you think about using `projectToDelete` state only to keep track the open/close state of the dialog ? We can open DeleteProjectDialog if `projectToDelete` is defined and reset it to undefine when closing the dialog

Reply:
There's a little issue using this ``projectToDelete`` for that purpose.  Check that answer here https://github.com/perses/perses/pull/916#discussion_r1085090443

Reply:
I see, sorry missed your reply earlier! It's fine to leave it as is then